open_academy/views/session_views.xml

```diff
@@ -16,6 +16,9 @@
         <field name="model">session</field>
         <field name="arch" type="xml">
             <form>
+                <header>
+                    <button id="wizard_session_button" name="%(wizard_session_action)d" string="Add Attendees" class="btn-primary" type="action"/>
```

Comment:
Could you check please if the name is taken from the function name please?

I mean remove the string and check if it takes the add_attendees method name instead

Reply:
do you mean that I change the name attribute for `name="add_attendees"` and also change the type for `type="object"`?

Reply:
I tested what you suggested and it doesn't appear